### PR TITLE
Create Mentorship Guidelines.md

### PR DESCRIPTION
**Pull request by Chinonyemildred**
I figured this markdown was supposed to be a separate file instead of adding the commit to the #1  PR. I've also aligned the image as I suggested in the home PR. What do you think?

### CODE REVIEW

File: Mentorship Guidelines.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<img src="https://user-images.githubusercontent.com/82679007/150870419-21f7d802-0e40-4a0c-b16a-17f37a18115c.png" alt="The Galaxy Mentor Network logo." width="400"  align="right" />
```
[the remark this answers is not in the document]
Comment (bebatut) [reply Mentorship Guidelines.md:1]: The logo would be likely in the  template for the pages. It does not need to be added in each Mardkown file

File: Mentorship Guidelines.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+# Galaxy Mentorship Network Guidelines
+
+Welcome to the Galaxy Community Mentorship Program designed to help community members who may be bioinformaticians, data analysts, students, programmers, e.t.c, become more proficient in the use of Galaxy tools and integration into the Galaxy Community.
```
Comment (bebatut) [inline Mentorship Guidelines.md:5]:
```suggestion
Welcome to the Galaxy Community Mentorship Program designed to help community members who may be bioinformaticians, data analysts, students, programmers, etc., become more proficient in the use of Galaxy tools and integration into the Galaxy Community.
```

File: Mentorship Guidelines.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+Highlights of this mentorship program include:
+
+- Week 1 - an introductory call between GMentees, GMentors, and Network Managers.
+- Week 4 - mid-program feedback
+- Week 8 - final feedback.
+
+## Program Tracking
+
+**How we track progress**
```
Comment (bebatut) [inline Mentorship Guidelines.md:19]: Should it be a subsection?

**Review by bebatut — COMMENTED** [review]
Thanks.
Can you rename the file to:
- avoid spaces (use `_` instead)
- use only lower case

File: Mentorship Guidelines.md (hunk 3 above)
Comment (Chinonyemildred) [reply Mentorship Guidelines.md:19]: Yes, it is...the same font as the paragraph but just bolder to show hierarchy. I'll appreciate tips on a more appropriate way to present it. Thank you:)

File: Mentorship Guidelines.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Chinonyemildred) [reply Mentorship Guidelines.md:1]: okay, thank you!

File: Mentorship Guidelines.md (hunk 3 above)
Comment (bebatut) [reply Mentorship Guidelines.md]: You could make it like proper subsection with `###` instead of the bold (same for other subsections after)

Comment (bebatut) [reply Mentorship Guidelines.md]: Then it would make the possibility to link to them

File: mentorship_guidelines.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+Mentees are members of the Galaxy Community who:
+
+- Are new to the community and help to become proficient in the use of Galaxy tools.
+- Want to learn how to contribute efficiently.
+- Want to transition from one part of the Galaxy ecosystem to another.
+- Need to learn how to be a mentor.
+- Need mentorship in carrying out their projects.
+
+
+### First Steps: Getting Started
```
Comment (bebatut) [inline mentorship_guidelines.md:47]: I would move this section just before `Mentors`

File: mentorship_guidelines.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+- The sole responsibility of your project rests on you, your mentor will not carry out your project for you.
+- Detailed and Timely reviews on your project or contributions will be given by GMentors.
+- At least 30 minutes of meeting calls between you and your mentor will be held weekly.
+
+
+## Mentors
+
+Mentors are members of the community who have volunteered to help mentor community members to foster the growth of the Galaxy Community.
+
+### First Steps: Getting Started
```
Comment (bebatut) [inline mentorship_guidelines.md:79]: As for the Mentees I would move that as last section

File: mentorship_guidelines.md (hunk 4 above)
Comment (Chinonyemildred) [reply mentorship_guidelines.md:47]: Do you mean the first steps sections?

Comment (bebatut) [reply mentorship_guidelines.md:47]: Yes

Comment (Chinonyemildred) [reply mentorship_guidelines.md:47]: Okay...all three roles have the same subsections starting with the first step(getting started) subsection. Moving the mentees' own down just before the `mentors` would require me to move that of the other roles too....to maintain consistency. Would that be okay?

Comment (bebatut) [reply mentorship_guidelines.md:47]: I thouht the same (see comments below) :smile:

Comment (Chinonyemildred) [reply mentorship_guidelines.md:47]: Great..I'll get to it now. Thank you! :)

**Comment by bebatut** [comment]
It looks good. I merge. Thanks

**Pull request merged**